### PR TITLE
Disable stress smoke test

### PR DESCRIPTION
**Pull request by sadhansood**
Using this PR to disable the smoke test to unblock people to land their PRs while we root cause the issue.

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ jobs:
           cargo nextest run --profile ci
       - name: benchmark (smoke)
         run: |
-          cargo run --package sui-benchmark --bin stress -- --log-path /tmp/stress.log --num-client-threads 10 --num-server-threads 24 --num-transfer-accounts 2 bench --target-qps 100 --num-workers 10  --transfer-object 50 --shared-counter 50 --run-duration 10s --stress-stat-collection
+          # cargo run --package sui-benchmark --bin stress -- --log-path /tmp/stress.log --num-client-threads 10 --num-server-threads 24 --num-transfer-accounts 2 bench --target-qps 100 --num-workers 10  --transfer-object 50 --shared-counter 50 --run-duration 10s --stress-stat-collection
           pushd narwhal/benchmark && fab smoke && popd
```
Comment (bmwill) [inline .github/workflows/rust.yml:95]: does this other line need to also be commented out?

Comment (mwtian) [reply .github/workflows/rust.yml:95]: This is not running the stress client so should be ok.

**Review by mwtian — APPROVED** [review]
I hope we can turn this back on soon. Otherwise we will spend more time to fix stress after it is broken.

**Comment by sadhansood** [comment]
> I hope we can turn this back on soon. Otherwise we will spend more time to fix stress after it is broken.

You are absolutely right @mwtian. I have identified the root cause, working on the fix and hopefully will uncomment this change out soon.

**Pull request merged**